lib/kino/config.ex

```diff
@@ -8,6 +8,11 @@ defmodule Kino.Config do
     Application.get_all_env(:kino) |> Keyword.take(@keys)
   end
 
+  @spec configuration(atom()) :: term()
+  def configuration(key, default \\ nil) do
```

Comment:
If you want to be double sure...

```suggestion
  def configuration(key, default \\ nil) when key in @keys do
```

But optional. :)